src/pyenphase/firmware.py

```diff
@@ -114,34 +118,30 @@ async def setup(self) -> None:
         if debugon:
             request_start = time.monotonic()
         try:
-            result = await self._get_info()
-        except httpx.TimeoutException as ex:
+            status_code, content = await self._get_info()
+        except asyncio.TimeoutError as ex:
             raise EnvoyFirmwareFatalCheckError(
                 500, "Timeout connecting to Envoy"
             ) from ex
-        except httpx.ConnectError as ex:
+        except aiohttp.ClientConnectorError as ex:
             raise EnvoyFirmwareFatalCheckError(
                 500, "Unable to connect to Envoy"
             ) from ex
-        except httpx.HTTPError as ex:
+        except aiohttp.ClientError as ex:
             raise EnvoyFirmwareCheckError(
                 500, "Unable to query firmware version"
             ) from ex
 
-        if (status_code := result.status_code) == 200:
+        if status_code == 200:
             if debugon:
                 request_end = time.monotonic()
-                content_type = result.headers.get("content-type")
-                content = result.content
                 _LOGGER.debug(
-                    "Request reply in %s sec from %s status %s: %s %s",
+                    "Request reply in %s sec from %s status %s",
                     round(request_end - request_start, 1),
                     self._url,
                     status_code,
-                    content_type,
-                    content,
                 )
```

Comment:
This no longer shows the /info content in debug. Content_type is not needed but content itself is useful to see.
```suggestion
                _LOGGER.debug(
                    "Request reply in %s sec from %s status %s: %s",
                    round(request_end - request_start, 1),
                    self._url,
                    status_code,
                    content,
                )
```